chapter_06/lib/services/plan_services.dart

```diff
@@ -28,7 +28,10 @@ class PlanServices {
   }
 
   void addTask(Plan plan, String description) {
-    final id = plan.tasks.last?.id ?? 0 + 1;
```

Comment:
Initially, I had this conflict `The receiver can't be null, so the null-aware operator '?.' is unnecessary.`. However, if I remove `?.` before `id`, then I get `The left operand can't be null, so the right operand is never executed.`. After that, the suggested looks like this `final id = plan.tasks.last.id;`. However, it breaks my code, so I actually came up with the above code.

Reply:
have you found any solution to this issue?

Reply:
lines 31 to 34 are the solution I came up with